{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python3-pytest-rerunfailures",
	"source": "pytest-rerunfailures",
	"version": "10.2-1",
	"section": "universe/python",
	"priority": "optional",
	"architecture": "all",
	"depends": [
		"python3-pkg-resources",
		"python3-pytest (>= 5.3)",
		"python3:any"
	],
	"installedSize": 51,
	"maintainer": "Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>",
	"description": "pytest plugin to re-run tests to eliminate flaky failures",
	"homepage": "https://github.com/pytest-dev/pytest-rerunfailures",
	"filename": "pool/universe/p/pytest-rerunfailures/python3-pytest-rerunfailures_10.2-1_all.deb",
	"size": 11814,
	"md5": "9fd41e8528417197d956e0102f12e85b",
	"sha1": "1540228857d671e2acb76e016a3fd9fc6de8432e",
	"sha256": "57ad2288f5a40c9300269937d59862ad9929e22eb4e3f8fd35a09ce6f0c5fb3b",
	"sha512": "86c5d17272714dfecbe1d562861d7e3827f0a0a43666b28b33632842f5c23a1c0e3f01d8102c8b9a6a77215e9a1d1fd547e2de6f7c1c71af3ae76be7efb295df",
	"descriptionMd5": "b39ddd4149884d1e62c3622245de3f54"
}